WalletWasabi/Tor/Http/IRelativeHttpClient.cs

```diff
@@ -25,9 +28,16 @@ Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToke
 			return SendAsync(request, DefaultIsolateStream, token);
 		}
 
+		/// <summary>
+		/// Sends an HTTP(s) request.
+		/// <para>The method can be used only when <see cref="DestinationUriAction"/> is set.</para>
+		/// </summary>
+		/// <exception cref="ArgumentNullException">If <see cref="DestinationUriAction"/> is <c>null</c>.</exception>
 		async Task<HttpResponseMessage> SendAsync(HttpMethod method, string relativeUri, HttpContent? content = null, CancellationToken cancel = default)
 		{
-			var requestUri = new Uri(DestinationUriAction.Invoke(), relativeUri);
+			var destinationUriAction = Guard.NotNull(nameof(DestinationUriAction), DestinationUriAction);
```

Comment:
Please do not provide default implementations in interfaces.

Reply:
Why?